go/test/endtoend/vreplication/cluster_test.go

```diff
@@ -87,11 +89,29 @@ type ClusterConfig struct {
 	vreplicationCompressGTID bool
 }
 
+func (cc *ClusterConfig) compressGTID() func() {
+	cc.vreplicationCompressGTID = true
+	return func() {
+		cc.vreplicationCompressGTID = false
+	}
+}
+
+func setVTTabletExperimentalFlags() func() {
```

Comment:
IMO a better name is ~ `enableAllVReplicationExperimentalFlags()` and a short comment would be helpful.

Reply:
done